src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -208,6 +209,8 @@ public Thickness BorderThickness
             }
         }
 
+        public double PrimaryScreenRenderScaling => Screen.AllScreens.FirstOrDefault(screen => screen.Primary)?.PixelDensity ?? 1;
```

Comment:
I wonder if this should be window specific? not just the first monitor? @kekekeks @maxkatz6

Reply:
It seems to be intended "the border is calculated with the DPI scaling of the actual monitor. But the AdjustWindowRectEx method **is not DPI aware so we need to calculate it with the DPI of the Primary screen**."